app/components/layout/top-nav.tsx

```diff
@@ -1,240 +1,104 @@
-import { Form } from '@remix-run/react';
+import React, { useState } from 'react';
 
```

Comment:
As I remember, we used code for the NavBar from https://tailwindui.com/components/application-ui/navigation/navbars. Instead of starting from scratch, why don't we use the same component and select React from the dropdown to leverage the benefits of Headless UI?)